### PR TITLE
worker: Extract function mkResonse to eliminate duplicate code

### PR DESCRIPTION
**Pull request by suzaku**
### What problem does this PR solve? 

Eliminate some duplicate code reported by `dupl`.

### What is changed and how it works?

Refactor by extracting a function `mkResponse`.

### Check List 

Tests 

 - Integration test

Code changes

Side effects

Related changes

### CODE REVIEW

File: dm/worker/server.go [hunk 1]
```diff
@@ -460,15 +422,20 @@ func (s *Server) QueryWorkerConfig(ctx context.Context, req *pb.QueryWorkerConfi
...
 	log.Infof("[server] receive MigrateRelay request %+v", req)
 
+	err := s.worker.MigrateRelay(ctx, req.BinlogName, req.BinlogPos)
+	if err != nil {
+		log.Errorf("[worker] %v MigrateRelay error %v", req, errors.ErrorStack(err))
+	}
+	return mkResponse(err), nil
+}
+
+func mkResponse(reqErr error) *pb.CommonWorkerResponse {
```
Comment (amyangfei) [inline dm/worker/server.go]: is `mkCommonWorkerResponse` more accurate ?

Comment (kennytm) [reply dm/worker/server.go]: Please spell out "make" instead of "mk".

**Comment by GregoryIan** [comment]
CI was broken  @suzaku

**Comment by amyangfei** [comment]
> CI was broken @suzaku

seems broken caused by log flushed slow issue, we have added log flush wait mechanism in test for the syncer unit status check, but no wait mechanism for loader unit status check

**Pull request merged**